wsmaster/codenvy-hosted-system-ram/src/main/java/com/codenvy/service/systemRam/SystemRamService.java

```diff
@@ -0,0 +1,51 @@
+/*
+ *  [2012] - [2016] Codenvy, S.A.
+ *  All Rights Reserved.
+ *
+ * NOTICE:  All information contained herein is, and remains
+ * the property of Codenvy S.A. and its suppliers,
+ * if any.  The intellectual and technical concepts contained
+ * herein are proprietary to Codenvy S.A.
+ * and its suppliers and may be covered by U.S. and Foreign Patents,
+ * patents in process, and are protected by trade secret or copyright law.
+ * Dissemination of this information or reproduction of this material
+ * is strictly forbidden unless prior written permission is obtained
+ * from Codenvy S.A..
+ */
+package com.codenvy.service.systemRam;
+
+import com.codenvy.service.systemRam.dto.RamInfo;
+
+import org.eclipse.che.api.core.ServerException;
+import org.eclipse.che.api.core.rest.Service;
+
+import javax.inject.Inject;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.core.MediaType;
+
+import static org.eclipse.che.dto.server.DtoFactory.newDto;
+
+/**
+ * Defines system ram REST API.
+ *
+ * @author Igor Vinokur
+ */
+@Path("/ram")
```

Comment:
i don't think __/api/ram/info__ is really good path, 
i would probably change it to something like __/api/system/ram__ or maybe __/api/system/raminfo__
@skabashnyuk what do you think about it?